keyboards/xd75/keymaps/jarred/keymap.c

```diff
@@ -0,0 +1,40 @@
+#include QMK_KEYBOARD_H
+
+#define _QW 0
+#define _LW 1
+#define _NV 2
+#define _NM 3
+
+#define _______ KC_TRNS
```

Comment:
Lines 8 and 9 can be safely deleted. QMK includes these definitions automatically with every keymap.

https://github.com/qmk/qmk_firmware/blob/4318797d198b58bb807b3e436c7d8924d8b4a6fe/quantum/quantum_keycodes.h#L29-L31